runtime/android/core_shell/BUILD.gn

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//build/config/android/rules.gni")
+import("//third_party/icu/config.gni")
+
+android_apk("xwalk_core_shell_apk") {
+  apk_name = "XWalkCoreShell"
+  android_manifest = "AndroidManifest.xml"
+  native_libs = [
+    "$root_out_dir/libxwalkcore.so",
+    "$root_out_dir/libxwalkdummy.so",
+  ]
+  deps = [
+    ":xwalk_core_shell_apk_assets",
+    ":xwalk_core_shell_apk_java",
+    ":xwalk_core_shell_apk_pak",
+    ":xwalk_core_shell_apk_resources",
+    "//base:base_java",
+    "//third_party/android_tools:android_support_v13_java",
```

Comment:
I don't think this one is necessary here, it's `xwalk_core_shell_apk_java` that uses it.